test/apiClients/AirtableApiClient.ts

```diff
@@ -1,5 +1,74 @@
 import test from 'ava';
+import sinon from 'sinon';
+import AirTableApiClient, { Organization } from '../../src/apiClients/AirTableApiClient';
+//import env from '../../src/env';
 
-test('Returns one page of results properly', t => {
-  t.true(true, 'replace this test');
+test('Returns one page of results properly', async t => {
+  const airtableApiClient: AirTableApiClient = new AirTableApiClient('keyzxy');
+  const eachPageStub = sinon.stub().returns(records);
+  sinon.stub(airtableApiClient.base, 'select').returns({ eachPage: eachPageStub });
```

Comment:
I thought we were using `nock` here. This test isn't very meaningful if we just stub the entire `eachPage` function, since it doesn't test your callback